### ISSUE TITLE
Difficult to tell what element in DashboardAppBar currently has focus

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/my-learning-analytics#803]
Thank you for contributing to this project!

- Make sure to search the [issues](https://github.com/tl-its-umich-edu/my-learning-analytics/issues) for duplicates first!

## Expected behavior (A clear and concise description of what you expected to happen) : 

When using the keyboard (Tab keyboard button) to navigate through links and options on the `DashboardAppBar` component, users should be able to tell visually what has focus and be able to select all options using Enter keyboard button.

## Describe the bug (Tell us what happens instead of the expected behavior) :

From the 10/28/19 accessibility review (which refers to the `DashboardAppBar` as a banner):

"Banner/title links have focus issues. It is hard to tell what has focus currently. See [https://www.mivideo.it.umich.edu/media/t/1_4cbw4s7i](https://www.mivideo.it.umich.edu/media/t/1_4cbw4s7i) ... Some users will not be able to operate the banner/title links with confidence, and will learn only by trial and error."

In addition, it may not be possible to use either the "My Learning Analytics" or "[Course]" link in this component by using the Enter button. 

## Additional context (Add any other context about the problem here) :

The accessibility review provides the following guidance:

"Current focus depends on a combination of [Material Design] focus treatment and native browser treatment (in Chrome a very unhelpful blue halo with does not play well with #00274c, the bane of our existence). Explicitly adding some CSS on thething:focus will deal with this."

Regarding the links to the landing page (the `CourseList` component, or `/courses/` route) and the course page (`/courses/XXXXX` route), segmenting these with additional Material UI components may enable traversing and selecting these separately using the keyboard.

**Comment by jennlove-um** [comment]
Testing passes: I can tab from one link to the next in the header and see the pulsing light blue outline on the MyLA home and course home links.

**Comment by jennlove-um** [comment]
I'll create a new issue for this.  Perhaps something has changed with the move to the LTI.
